lib/ui/widgets/digitalposition.js

```diff
@@ -97,6 +117,14 @@ var positionFormats = [
       label: "DDirM'S''ddd",
       fontSize: 15
     }
+  },
+  function(positionObject) {
+    return {
+      latitude: displayDDM(positionObject.latitude, coordTypeLatitude, false),
+      longitude: displayDDM(positionObject.longitude, coordTypeLongitude, false),
+      label: "Dir D\u00b0 M.mmm'",
```

Comment:
As you display the position in this format `59° 43.2601' N` change the label to: `D\u00b0 M.dddd' Dir`
You can also display like that to save one chr `59° 43'2601 N` and label to: `D\u00b0 M'dddd Dir`

Reply:
My goal here was to match OPNs "Degrees, decimal minutes", as this format is commonly used here. 
I've corrected the label to D\u00b0 M.mmmm' Dir